remotespark/livyclientlib/livysession.py

```diff
@@ -90,11 +89,14 @@ def create_sql_context(self):
         self.ipython_display.writeln("Creating HiveContext as 'sqlContext'")
         command = self._get_sql_context_creation_command()
         try:
-            command.execute(self)
+            (success, out) = command.execute(self)
         except LivyClientTimeoutError:
             raise LivyClientTimeoutError("Failed to create the SqlContext in time. Timed out after {} seconds."
                                          .format(self._wait_for_idle_timeout_seconds))
-        self.created_sql_context = True
+        if success:
```

Comment:
You just modified the behavior of livysession (it can now raise a ValueError). What class consumes this exception? I think it should be the magics and should set a fatal error as True. Is that already happening here?

For this to hold True, all intermediate classes between livysession and kernelmagics must not catch any ValueErrors in the methods that get called. Is that already True? Can we unit test that?